mergify_engine/rules/parser.py

```diff
@@ -153,13 +165,27 @@ def _iso_datetime(tokens: typing.List[pyparsing.Token]) -> datetime.datetime:
         )
     }
 )
-_time_range = (_match_time + pyparsing.Literal("-") + _match_time).setParseAction(
-    lambda toks: {
+
+
+def _parse_time_range(toks: typing.List[pyparsing.Token]) -> typing.Any:
+    time1 = toks[0]
+    time2 = toks[2]
+
+    # NOTE(sileht): In case of the format is `10:00-18:00[Europe/Paris]`,
```

Comment:
Does this mean you can have:

```
10:00[Europe/Paris]-19:00[US/New York]
```
?

Reply:
yes

Reply:
Maybe write it down in the documentation? 🤔